src/api/test-controller/index.js

```diff
@@ -54,10 +60,7 @@ import assertRequestHookType from '../request-hooks/assert-type';
 import { createExecutionContext as createContext } from './execution-context';
 import { isClientFunction, isSelector } from '../../client-functions/types';
 
-import {
-    MultipleWindowsModeIsDisabledError,
-    MultipleWindowsModeIsNotAvailableInRemoteBrowserError
-} from '../../errors/test-run';
+import { MultipleWindowsModeIsDisabledError, MultipleWindowsModeIsNotAvailableInRemoteBrowserError } from '../../errors/test-run';
```

Comment:
Maybe we can enable `consistent: true` to allow separate long imports even when their number is below the critical value